src/libs/tosService.js

```diff
@@ -0,0 +1,55 @@
+import ReactMarkdown from 'react-markdown';
+import DOMPurify from 'dompurify';
+import React from 'react';
+import {ToS} from './ajax';
+import homeHeaderBackground from "../images/home_header_background.png";
+
+
+export const TosService = {
+
+  getBackgroundStyle: () => {
+    return {
+      marginTop: '-50px',
+      paddingTop: '25px',
+      minHeight: '700px',
+      backgroundImage: `linear-gradient(to right, transparent, white 50%), url(${homeHeaderBackground})`,
+      backgroundRepeat: 'no-repeat'
+    };
+  },
+
+  getContainerStyle: () => {
+    return {
+      margin: '50px',
+      maxWidth: '800px',
```

Comment:
I was curious about how we decide on the standard width for containers like this one. When I get to the ToS page my browser window is larger than the container width but it doesn't scale up after the max value. It feels a little weird to me when I resize my window so I was just wondering.

Reply:
I don't think we have a standard for this type of thing. The intent here was to try and create an outer container that would be large enough to display the inner, scrollable content with some comfortable spacing.